spec/models/pairing_spec.rb

```diff
@@ -2,7 +2,6 @@
 
 RSpec.describe Pairing, type: :model do
   describe 'relationships' do
-    it { should belong_to :pairee }
```

Comment:
Curious about this part: did you take it out because it made the relationship required?

If so, there's an option you can pass to `belong_to` which makes the relationship optional; that way `Pairing` could still have a `pairee` instance method.

Reply:
```ruby
it { should belong_to(:coupon).optional }
```